wachter/actions.py

```diff
@@ -8,24 +8,25 @@
 import constants
 import re
 import random
+from typing import *
 
 logging.basicConfig(format='%(asctime)s - %(name)s - %(levelname)s - %(message)s',
                     level=logging.INFO)
 logger = logging.getLogger(__name__)
 
 
-def on_error(bot, update, error):
+def on_error(bot: telegram.Bot, update: telegram.Update, error: telegram.error.TelegramError):
```

Comment:
IMO it would be less wordy (and still clear for the reader) to import `Bot`, `Update`, etc from `telegram` and use them here without the `telegram.` part.

Reply:
Also could you clean the L4-5 imports:
```
import telegram
from telegram import InlineKeyboardButton, InlineKeyboardMarkup
```

It might be better to have (for example)
```
from telegram import Bot, InlineKeyboardButton, ...
from telegram.ext import JobQueue, ...
```

Reply:
done!